code/inetfile/chttpget.cpp

```diff
@@ -223,14 +223,15 @@ void ChttpGet::WorkerThread()
 	if(strnicmp("HTTP/",p,5)==0)
 	{
 		char *pcode;
-		pcode = strchr(p,' ')+1;
-		if(!pcode)
+		pcode = strchr(p,' ');
+		if(pcode == null)
```

Comment:
FYI, `null` is not valid in C++. The correct keyword is `nullptr`.

Reply:
Thank you